### ISSUE TITLE
camera can not initial

### ISSUE THREAD

**Issue by wangnan31415926** [NVIDIA-AI-IOT/jetcam#28]
Python 3.6.9 (default, Oct  8 2020, 12:12:24) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from jetcam.csi_camera import CSICamera
>>> camera = CSICamera(width=224, height=224, capture_width=1080, capture_height=720, capture_fps=30)
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/jetcam-0.0.0-py3.6.egg/jetcam/csi_camera.py", line 24, in __init__
RuntimeError: Could not read image from camera.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/dist-packages/jetcam-0.0.0-py3.6.egg/jetcam/csi_camera.py", line 27, in __init__
RuntimeError: Could not initialize camera.  Please see error trace.

**Comment by wangnan31415926** [comment]
USE THIS 'nvarguscamerasrc sensor-id=%d ! video/x-raw(memory:NVMM), width=%d, height=%d, format=(string)NV12, framerate=(fraction)%d/1 ! nvvidconv ! video/x-raw, width=(int)%d, height=(int)%d, format=(string)BGRx ! videoconvert ! video/x-raw, format=(string)BGR ! appsink' TO change CSI_CAMERA.PY
It will be work!!

**Comment by Liamkk** [comment]
It had showen the same error

**Comment by KnechtNoobrecht** [comment]
> USE THIS 'nvarguscamerasrc sensor-id=%d ! video/x-raw(memory:NVMM), width=%d, height=%d, format=(string)NV12, framerate=(fraction)%d/1 ! nvvidconv ! video/x-raw, width=(int)%d, height=(int)%d, format=(string)BGRx ! videoconvert ! video/x-raw, format=(string)BGR ! appsink' TO change CSI_CAMERA.PY It will be work!!

I have the same problem, but don't understand your answer. Can you please elaborate further?
